src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -4387,6 +4386,28 @@ else if (responseText.contains("a horrible grinning clown head emerges")) {
     }
   }
 
+  private static void trackEnvironment(final KoLAdventure location) {
+    var environment = location != null ? location.getEnvironment() : "none";
+
+    var symbol =
+        switch (environment) {
+          case "outdoor" -> "o";
+          case "indoor" -> "i";
+          case "underground" -> "u";
+          case "underwater" -> "x";
+          default -> "?";
```

Comment:
I think default (e.g. as will appear for "none") should be "x" same as underwater is "x".

Reply:
I want to track default separately for when new locations are added. I'm adding some functions to predict your current pill which take the ? into account smartly.

Reply:
looks like I was wrong about this anyway -- by the forum thread, zoneless combats don't count towards Fleshazole, but I thought they did